tests/integration/graviton_test.py

```diff
@@ -310,7 +310,7 @@ def test_stable_teal_generation(subr, mode):
         },
     },
     slow_fibonacci: {
-        "inputs": [(i,) for i in range(18)],
+        "inputs": [(i,) for i in range(17)],
```

Comment:
Have to tick down the input size by 1, for input == 17, frame based subroutine doesn't reach cost of 70000, which is conflicting with scratch slot based implementation.

Reply:
would you mind bringing back the case of 17, but then adding an ugly (but for a test ugliness is not such a big deal) work around that does something like the following starting on line 567:

```python
    if subr.name() == "slow_fibonacci":
        assert len(inspectors6) == len(inspectors8) == 18
        del inspectors6[-1]
        del inspectors8[-1]
    ...
```

Reply:
That I can do, I was just quite unaware of how it gets around the identity invariant validation.

Reply:
And yes, it is for `inspector8`, on the final case, doesn't blow up the budget 70k line.